src/webui/www/private/views/preferences.html

```diff
@@ -648,10 +648,12 @@
         <legend>QBT_TR(Language)QBT_TR[CONTEXT=OptionsDialog]</legend>
         <label for="locale_select">QBT_TR(User Interface Language:)QBT_TR[CONTEXT=OptionsDialog]</label>
         <select id="locale_select">
+            <option value="az@latin">azərbaycan dili</option>
             <option value="en">English</option>
             <option value="en_AU">English(Australia)</option>
             <option value="en_GB">English(United Kingdom)</option>
             <option value="eo_EO">Esperanto</option>
+            <option value="et">eesti, eesti keel</option>
```

Comment:
Just to be sure, most of the locale names start with uppercase but the newly added ones do not. Are they correct?

Reply:
Yes, both are correct if it's a name of the language, not a specific country. If it is about language name written in Estonian, then we use non capital letters. Estonian language
 institute and Microsoft Windows 10 language
 pack information websites can be checked to verify.

Reply:
@Chocobo1 I will capitalize the names for languages with latin chars.